src/gps/gpsd_test.cpp

```diff
@@ -30,79 +30,79 @@ static void libgps_dump_state(struct gps_data_t *collect)
 {
     /* no need to dump the entire state, this is a sanity check */
 #ifndef USE_QT
-    /* will fail on a 32-bit macine */
+    /* will fail on a 32-bit machine */
     (void)fprintf(stdout, "flags: (0x%04x) %s\n",
-		  (unsigned int)collect->set, gps_maskdump(collect->set));
+				  (unsigned int)collect->set, gps_maskdump(collect->set));
 #endif
     if (collect->set & ONLINE_SET)
-	(void)fprintf(stdout, "ONLINE: %lf\n", collect->online);
+		(void)fprintf(stdout, "ONLINE: %ld.%09ld\n", collect->online.tv_sec, collect->online.tv_nsec);
```

Comment:
This is causing a compile error for me . . . maybe this is from a newer version of libgpsmm.h? I reverted it in #117 but if the tests fail I guess that means I need to update my local version of libgpsmm